.env.example

```diff
@@ -1 +0,0 @@
-NEXT_PUBLIC_BACKEND_URL=
```

Comment:
We should keep `.env.example`
Good way to communicate if you're missing an env var